nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -103,12 +103,12 @@ def do_compression(
                 quantized_nodes_ids.add(id(weight_node))
 
         internal_weight_params = all_weight_params
-        if mode != CompressWeightsMode.INT8:
+        if mode not in [CompressWeightsMode.INT8_SYM, CompressWeightsMode.INT8_ASYM]:
             internal_weight_params = list(filter(lambda wp: wp.metatype != OVEmbeddingMetatype, all_weight_params))
             if not is_last_layer_compressed:
                 internal_weight_params = internal_weight_params[:-1]
-            primary_config = WeightCompressionConfig(mode=mode, group_size=group_size)
-            _assign_mixed_precision(internal_weight_params, ratio, primary_config)
+        primary_config = WeightCompressionConfig(mode=mode, group_size=group_size)
+        _assign_mixed_precision(internal_weight_params, ratio, primary_config)
```

Comment:
Could please refactor this part to make it more clear?
- `_assign_mixed_precision` is not actually about mixed precision when ratio=1, I'd move outside this logic
- internal weight params and if condition are needed for mixed precision only

BTW, which mode is assigned for embeddings in INT4_ASYM / INT4_SYM? Is it always INT8_ASYM? I'd mention this in docs.